src/varlist.h

```diff
@@ -35,11 +35,6 @@ varstrx(VAR_LC_CTYPE,	"LC_CTYPE",	NULL,		ch_locale)
 varstrx(VAR_LC_TIME,	"LC_TIME",	NULL,		ch_locale)
 #endif /* HAVE_SETLOCALE */
 varstr (VAR_MAIL,	"MAIL",		NULL,		ch_mailfile)
-#if ENABLE_OPTION102
```

Comment:
Did you want to remove this?

Reply:
Yeah, I actually moved it, it's just showing up weird I guess. I confirmed and it's ok in the file.

There was no reason for it to be under VAR_OPTION102, so it's now under VAR_option102.